android-components/components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -43,17 +43,19 @@ import mozilla.components.ui.icons.R as iconsR
 /**
  * Initializes and resets the Toolbar for a Custom Tab based on the CustomTabConfig.
  *
- * @param toolbar Reference to the browser toolbar, so that the color and menu items can be set.
- * @param sessionId ID of the custom tab session. No-op if null or invalid.
- * @param menuBuilder Menu builder reference to pull menu options from.
- * @param menuItemIndex Location to insert any custom menu options into the predefined menu list.
- * @param window Reference to the window so the navigation bar color can be set.
- * @param shareListener Invoked when the share button is pressed.
- * @param closeListener Invoked when the close button is pressed.
- * @param updateToolbarBackground Whether or not the toolbar background should be changed based on
- * [CustomTabConfig.toolbarColor].
- * @param forceActionButtonTinting When set to true the toolbar action button will always be tinted
+ * @property store The given [BrowserStore] to use.
+ * @property toolbar Reference to the browser toolbar, so that the color and menu items can be set.
```

Comment:
```suggestion
 * @property toolbar Reference to the [BrowserToolbar], so that the color and menu items can be set.
```
If we could use the class reference in-place of the actual naming, I think that would be an improvement since if things are named well, we would basically say the same thing verbatim with the ability to jump to definition when reading the kDocs.

This also applies to your other kDocs where applicable.